src/main/webapp/tutorials.html

```diff
@@ -28,28 +30,25 @@ <h3> Watch the video to learn how to use this page </h3>
     </div>
 
     <div id="content-right">
+
       <div id="suggestion-form">
         <div id="suggestion">
           <h4>Create a new suggestion</h4>
-          <form action="/new-suggestion" method="POST">
-            <label for="platform">Platform (ex: Zoom, Facebook etc.):</label><br>
-            <input type="text" id="platform" name="platform" placeholder="Your platform suggestion ..."><br>
-            <label for="addition">Additional details &amp; specific questions:</label><br>
-            <textarea rows="5" cols="30" id="addition" name="addition" placeholder="Insert additional details ..."></textarea>
-          </form>
+          <label for="platform">Platform (ex: Zoom, Facebook etc.):</label><br>
+          <textarea rows="1" cols="30" id="platform" name="platform" placeholder="Your platform suggestion ..."></textarea><br>
+          <label for="addition">Additional details &amp; specific questions:</label><br>
+          <textarea rows="5" cols="30" id="addition" name="addition" placeholder="Insert additional details ..."></textarea><br>
+          <button onclick="makeSuggestion();" value="Submit" >Submit</button>
```

Comment:
See comment about setting onclick attribute in Javascript.

Reply:
added getElementById('make-suggestion')